docs/csharp/programming-guide/classes-and-structs/toc.yml

```diff
@@ -7,41 +7,42 @@
     href: objects.md
   - name: Structs
     href: structs.md
-    items:
-    - name: Using Structs
-      href: using-structs.md
+  - name: Using Structs
+    href: using-structs.md
   - name: Inheritance
     href: inheritance.md
   - name: Polymorphism
-    href: polymorphism.md
     items:
+    - name: Overview
+      href: polymorphism.md
     - name: Versioning with the Override and New Keywords
       href: versioning-with-the-override-and-new-keywords.md
     - name: Knowing When to Use Override and New Keywords
       href: knowing-when-to-use-override-and-new-keywords.md
     - name: "How to: Override the ToString Method"
       href: how-to-override-the-tostring-method.md
-  - name: Abstract and Sealed Classes and Class Members
-    href: abstract-and-sealed-classes-and-class-members.md
+  - name: Members
```

Comment:
Instead of grouping these together, shouldn't we simply expand where the repetition happens to include an overview link?

Reply:
I left them as a single group, but re-ordered them to be more logical.